docusaurus/docs/iOS/controllers/channels.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Channel Controllers
+---
+
+## ChannelListController
+
+`ChatChannelListController` allows you to observe a list of chat channels based on the provided query and to paginate channels.
```

Comment:
Link to more information about pagination.

Reply:
good point, I dont want to document controllers here since it's mainly an LLC thing but adding something to pagination makes sense (not a code example though, that's way too much code ;))